wallet/createtx.go

```diff
@@ -164,11 +165,13 @@ func createTx(
 	bs *waddrmgr.BlockStamp,
 	feeIncrement btcutil.Amount,
 	mgr *waddrmgr.Manager,
-	changeAddress func(*waddrmgr.BlockStamp) (btcutil.Address, error)) (
+	changeAddress func(*waddrmgr.BlockStamp) (btcutil.Address, error),
```

Comment:
Not sure what happened here with this function having one parameter per line, but this doesn't match the style of the rest of the code.  I realized you just followed suit, but now that I'm reviewing this and see it, if you would, please go ahead and fix the other params too so it's consistent with the rest of the code base.